.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "app/RubyCocoa"]
+	path = app/RubyCocoa
+	url = https://github.com/alloy/rubycocoa.git
```

Comment:
The gist of it is that RubyCocoa has clearly mostly been used to implement Cocoa stuff in Ruby, there’s some lack when interfacing with pure Ruby from the ObjC side.